recipes-core/packagegroups/packagegroup-arago-tisdk-sourceipks-sdk-host.bb

```
@@ -43,24 +43,21 @@ UTILS:append:ti33x = " \
     omapconf-src \
     pru-icss-src \
     mmwavegesture-hmi-src \
-    evse-hmi-src \
+    evse-hmi-src \ 
```

```suggestion
    evse-hmi-src \
```

Seems like a white space was added unintentionally